airbyte-integrations/connectors/destination-mssql/src/main/java/io/airbyte/integrations/destination/mssql/SqlServerOperations.java

```diff
@@ -53,7 +53,7 @@ public String createTableQuery(String schemaName, String tableName) {
             + "WHERE s.name = '%s' AND t.name = '%s') "
             + "CREATE TABLE %s.%s ( \n"
             + "%s VARCHAR(64) PRIMARY KEY,\n"
-            + "%s VARCHAR(MAX),\n"
```

Comment:
@etsybaev can you leave a comment in the code describing why NVARCHAR is used since we typically use VARCHAR?

Reply:
@sherifnada done. Added comment to code and documentation. Thanks